packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,229 @@
+Reading by column from top to bottom is the more common way information is consumed visually, so when presenting this information by row between the two columns it might be confusing for assistive technology users and users in general.
+- When passing in Clinical Item Displays, keep in mind that only 8 can be used within a view at a time. Any additional displays passed in will be ignored.
+- Displays are also split between two columns with every other display being moved to the right hand column.
+
+----
+
+### Code Considerations
+
+#### Headings
+
+<Notice variant="important" ariaLevel="3">
+
+It is **strongly** recommended to add a heading element above the implementation of the Clinical Item View component.
+**A header not only acts as a label for the information being communicated to the end users, but also helps with visual and screenreader navigation throughout a page.**
+The recommended way to implement this header is to utilize [Terra Heading](https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/heading/about)
+or the [Terra Clincial Header](https://engineering.cerner.com/terra-clinical/components/terra-clinical-header/clinical-header/clinical-header).
+
+</Notice>
+
+The Clinical Item View structure consists of an unordered list `<ul>` filled with list items. It is vital to include a header to ensure that the list of displays have proper context within the webpage.
+For example - if an Item View contains contact information for a patient, a header should be added above the Item View with a label such as 'Contact Information' or 'Patient Contact Information'.
+
+Best practices for implementing headers are as follows:
+- Use one unique `<h1>` per page, above the main content it describes.
+- Use lower headings to describe the content below. Do not use an HTML heading just to make the text appear bigger or stand out.
+- Use heading levels like the index of a book: hierarchical.
+- Do not choose a heading by its size, but by its level in the context of the content.
+- Do not skip a heading level from the top down.
+
+More information on heading standards can be found [here](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/).
+
+Here are some examples of implementing a Terra Heading element above an Item View.
+
+##### Code Examples
+
+<ItemViewWithHeader
+    title="ItemView - With Header"
+/>
+<ItemViewTwoColumnWithHeader
+    title="ItemView - Two Column Layout With Header"
+/>
+
+#### True Columns vs By Row
+
+The two column layout for Clinical Item View has been updated to include an option for true columns instead of rows formatted to look like columns. This affects not only the programmatic layout but also the screenreader's behavior in how these displays are read.
+The implementation of this is accomplished behind the `trueColumn` prop, which defaults to false, but should be set to true for accessibility purposes.
```

Comment:
Since true/false here are values, let's format them as monospace. Same for anywhere else these are used as values.

Reply:
Sounds good, changed here: https://github.com/cerner/terra-clinical/pull/916/commits/545df7819c16c911464ded34b38055cf814b790f#diff-a09eb4a886b729c86a83daac740d9a14a92e66fe733a7a907899150b7fdae956R61